serverless.yml

```diff
@@ -27,4 +27,4 @@ functions:
     handler: src/handler.run
     timeout: 60
     events:
-      - schedule: rate(1 minute)
```

Comment:
One point of running this kind of stuff frequently is to retry the transaction soon in case we underprice the gas. From that perspective 10 minutes is risky.

Reply:
I think it's more likely that the 10 minute schedule will bite us due to gas price and RPC availability issues